packages/strapi-design-system/src/Checkbox/Checkbox.tsx

```diff
@@ -24,10 +23,18 @@ const CheckboxTick = (props) => {
   return <BaseCheckbox id={id} {...props} />;
 };
 
-export const Checkbox = ({ children, disabled, id, hint, error, ...props }) => {
+interface CheckboxProps {
+  children: React.ReactNode;
+  disabled?: boolean;
+  error?: string;
+  hint?: string | React.ReactNode | React.ReactNode[];
+  id?: string | number;
```

Comment:
these can all be brought from `FieldProps` if you `extend` this type

Reply:
ok thanks